Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -3,6 +3,8 @@
 using System.Collections.Generic;
 using UnityEngine.InputSystem.Utilities;
 
+////TODO: make the FindAction logic available on any IEnumerable<InputAction> and IInputActionCollection via extension methods
```

Comment:
Should we be moving the TODO code into somewhere where we can capture and track the tasks? Maybe favro/jira cards?

Reply:
From my POV, moving them anywhere other than the code is not an improvement. I found that the only TODOs/FIXMEs/etc. that I consistently see and am reminded of are the ones I constantly stumble over in the code. Whenever I have moved them to separate lists in the past, it was the first step towards having them become obsolete and outdated and having a list full of random no-longer-relevants that I would soon stop looking at :)

Also, other than these being in *my* face, they are there for *everyone* to see on GitHub instead of on a private board inside of Unity (which 2 years from now will for sure get deleted as part of us moving yet to yet another ticket tracking service).